UndertaleModTool/Editors/UndertaleCodeEditor.xaml.cs

```diff
@@ -679,40 +681,43 @@ private async Task DecompileCode(UndertaleCode code, bool first, LoaderDialog ex
 
                     if (decompiled != null)
                     {
-                        string[] decompiledLines;
+                        StringReader strReader;
+                        StringBuilder strBuilder;
```

Comment:
I like "decompiledLines" more as a variable name for this. `strBuilder` sounds too generic